# This service allows the user to change the camera frame that the vision 
#  system is using to determine the world coordinates of an object

float64[3] trans
float64[4] quat

string cameraName

---

int64 ret
string msg
